requirements-dev.txt

```diff
@@ -1,12 +1,8 @@
 aspy.yaml==0.3.0
-boto3==1.4.7
-cached-property==1.3.0
```

Comment:
So I wasn't 100% sure on removing `cached-property` and `identify` I couldn't find where they were used though so I figured it was boiler-plate.